src/provideCompletionItems.ts

```diff
@@ -37,6 +37,11 @@ async function completionsListFor(
       return [];
     }
 
+    if (document.lineAt(position.line).text.trim() === "") {
+      await runCompletion(document, position);
+      await sleep(110);
+    }
```

Comment:
this lets the binary "warm up" upon empty line completions before actually requesting completions